bsp/env/freedom-e300-hifive1/entry.S

```diff
@@ -1,55 +1,47 @@
-// See LICENSE for License Details
+// See LICENSE for license details
```

Comment:
Do you think that we should make (perhaps as a different PR) these files common in a shared/common directory vs. simlinking?